src/Views/IMEView.h

```diff
@@ -6,21 +6,15 @@
 #include <codecvt>
 #include <sys/socket.h>
 
-#include <global_include.h>
-#include <singleton.h>
-#include <Views/View.h>
-#include <activity.h>
+#include "../global_include.h"
```

Comment:
Why was it changed?

Reply:
You mean the whole file or just that line or what?

Reply:
I meant only this line. By the way, the `global_include.h` header was a bad idea of mine, it increases compilation times and it makes the interdependency unclear.

Reply:
Because this a local dependency (implemented in this project). https://stackoverflow.com/questions/21593/what-is-the-difference-between-include-filename-and-include-filename

Reply:
Oh I meant why is `global_include.h` now used instead of the relevant headers. For the `#include <>` part this is alright since https://github.com/devnoname120/vhbb/blob/master/CMakeLists.txt#L64. I used this before I had an IDE that could resolve paths automatically, so I suppose this is not necessary anymore.

Reply:
Well, refactoring everything to remove the `global_include.h` seems to be a lot of work. Maybe this should go into a new Issue?

Reply:
Ah indeed, I suppose it would be possible to copy the content of `global_include.h` in all the files where it's included (using a text replace) and then use CLion's `optimize imports` functionality.

Reply:
Yes, that sounds smart. But as said lets make a different issue and PR for that,